src/streamlink/plugins/rtlxl.py

```diff
@@ -1,24 +1,25 @@
 import re
+import json
 
 from streamlink.plugin import Plugin
 from streamlink.plugin.api import http, validate
 from streamlink.stream import HDSStream, HLSStream, RTMPStream
 
-_url_re = re.compile(r"""http(?:s)?://(?:\w+\.)?rtlxl.nl/#!/(?:.*)/(?P<uuid>.*?)\Z""", re.IGNORECASE)
-
+_url_re = re.compile("""http(?:s)?://(?:\w+\.)?rtl.nl/video/(?P<uuid>.*?)\Z""", re.IGNORECASE)
 
 class rtlxl(Plugin):
     @classmethod
     def can_handle_url(cls, url):
         return _url_re.match(url)
-
+			

```

Comment:
I should have requested this be modified to remove that extra space but whatever.